### DISCUSSION TITLE
Qiita CLI previewのhostを変更したい

### DISCUSSION THREAD

**Discussion by goosys** [increments/qiita-discussions#502] [要望・アイデア・不具合]
Qiita CLIの提供ありがとうございます。

ローカルマシンで使用するには問題なかったのですが、リモートマシンで使用する際にHOSTが変更できなくて詰まってしまいました。
ソースを拝見したところ、PORTの方は `process.env.QIITA_CLI_PORT` で変更できるようでしたので、HOSTの方も同様に可変できるようになると嬉しいです。
具体的には `127.0.0.1` や `0.0.0.0` に変更して使いたいです。

ご検討よろしくお願いいたします。

**Comment by acronhub** [comment]
@goosys 
ご意見ありがとうございます！

HOSTの変更について技術的に実装が難しい箇所があり、すぐには対応が難しいです。
今後対応できるか検討していきますので、ご理解いただけると幸いです :bow:

**Reply by goosys** [reply]
@acronhub ご回答ありがとうございます！
すぐには対応不可ということで了解いたしました。アップデートを気長に待ちたいと思います。 🙇‍♂️
